spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/client/GrpcClientAutoConfiguration.java

```diff
@@ -44,7 +44,7 @@ public class GrpcClientAutoConfiguration {
 	@Bean
 	@ConditionalOnMissingBean(GrpcChannelFactory.class)
 	public DefaultGrpcChannelFactory defaultGrpcChannelFactory(final List<GrpcChannelConfigurer> configurers,
-			ChannelCredentialsProvider credentials, GrpcClientProperties channels, SslBundles bundles) {
+			ChannelCredentialsProvider credentials, GrpcClientProperties channels, SslBundles ignored) {
```

Comment:
Eventhough we don't have checkstyles enforced in autoconfigure module yet on the command line. I have them setup in my IDE so when I go to commit it tells me all about it. These were harmless enough that I just piggybacked them here. Let me know if this is an issue and I can do it in subsequent commit or subsequent PR.